### DISCUSSION TITLE
Another Printer Kill Reason M112 Shutdown

### DISCUSSION THREAD

**Discussion by Ekstreme** [Jyers/Marlin#252] [Q&A] [answered]
I am using the pre-compiled firmware E3V2-ManualMesh-3x3-v4.2.2.bin no ABL installed 4.2.2 board. I attempted to start the manual bed levelling program but cancelled out before the first point. As soon as I did that the LCD locked up:

Printer Kill Reason
M112 Shutdown
Restart Required

Tried to reproduce but I can'y seem to get it to crash even after cancelling and completing the bed levelling program. It has been printing perfectly fine though I have gone back to manual levelling.

**Comment by Jyers** [comment, marked as answer]
Are you by any chance connected to a host device such as octoprint? That kill message indicates that something sent an M112 command to the printer, most likely because the host device thought there was a error.

**Reply by Ekstreme** [reply]
Why yes I did have Octoprint connected. Sorry for the false alarm.
